server/internal/infrastructure/db/sqlite/migration/20241015101830_remove-descriptor.down.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE vtxo DROP COLUMN addr;
```

Comment:
let's merge all migration files into a single one, we don't really care now about doing this politely.